detectors/node/opentelemetry-resource-detector-docker/README.md

```diff
@@ -0,0 +1,48 @@
+# OpenTelemetry Resource Detector for Docker
+
+[![NPM Published Version][npm-img]][npm-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+[component owners](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/.github/component_owners.yml): @abhee11
+
+Resource detector for docker.
```

Comment:
According to [the semantic conventions for resource container](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/container.md), the following 5 attributes belong to the `docker` resource:
```
container.name, container.id, container.runtime, container.image.name, container.image.tag
```

The current implementation only extracts a single value (`container.id`) of these 5 and only for `CGroupV1` (not sure how many other types are possible).

I suggest to make that clear in the README, or support the other fields in the spec

Reply:
currently in the readme it does say that we populate only container id - I will add more details in case if it is missing - thanks